src/components/FilmList/FilmList.tsx

```diff
@@ -0,0 +1,56 @@
+import "./style.scss";
+import { films } from "../../films";
+import { Film } from "../Film";
+import { FilmObj } from "../../interfaces";
+import ErrorBoundaries from "../ErrorBoundaries/ErrorBoundaries";
+
+interface FilmListProps {
+  genre: string;
+  searchFilm: string;
+  sort: string;
+}
+
+const FilmList: React.FC<FilmListProps> = ({ genre, searchFilm, sort }) => {
+  const filmes: any = films
+    .filter((el) => {
+      if (genre === "All") {
+        return true;
+      }
+      return el.genre.indexOf(genre) > -1;
+    })
+    .filter((el) => {
+      if (searchFilm === "") {
+        return true;
+      }
+
+      return el.name.toLowerCase().includes(searchFilm.toLowerCase());
+    })
+    .sort((a: FilmObj, b: FilmObj) => {
+      if (sort === "name") {
+        if (a.name < b.name) {
+          return -1;
+        }
+        if (a.name > b.name) {
+          return 1;
+        }
+        return 0;
+      } else {
+        return a.year - b.year;
+      }
+    })
+    .map((el: FilmObj, index: number) => {
+      return (
+        <ErrorBoundaries>
+          <Film film={el} key={index} />
+        </ErrorBoundaries>
+      );
+    });
```

Comment:
it is definitely not a good syntax and algorithm;
You always go through all movies even if genre === ALL or searchFilm is === ''.
1) rewrite it into three different functions, filter, sort, and render films
2) make more effective algorithm for filtering
3) create render function for film array

What is the reason to put ErrorBoundary for each film? 
if you want to catch it here (i think, that there is no reason, because you can put ErrorBoundary for all application in root component), put ErrorBoundary for array of films, not each film